src/screens/FeedScreen/Header.js

```diff
@@ -163,6 +207,11 @@ const _renderAnonimity = ({
           </View>
         </View>
       </View>
+      <BottomSheetMenu
+        refSheet={refSheet}
+        dataSheet={dataSheet}
+        height={isShowDelete ? 182 : 130}
+      />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BottomSheetMenu` component is added to the render method. This is necessary for displaying the bottom sheet menu.
```diff
+      <BottomSheetMenu
+        refSheet={refSheet}
+        dataSheet={dataSheet}
+        height={isShowDelete ? 182 : 130}
+      />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->